utils/headers/README.md

```diff
@@ -0,0 +1,11 @@
+<!--
+ Copyright Strimzi authors.
+ License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+-->
+
+# Headers
+
+This directory contains files with copyright/header statements which will be programmatically added to all source files. These headers are applied at build time to all built output, and at lint time via the `npm run lint:allfiles` script. Files in this folder should be:
```

Comment:
A little picky again but either drop the word `be` at the end of the sentence as it is included in the bullet points or drop the word off each of the bullet points to read correctly.

Reply:
Agreed this reads more nicely, will update in the next commit.